### ISSUE TITLE
Build of 0.17.0 fails: `tests/comments-2.command: line 1: ./initool: No such file or directory`

### ISSUE THREAD

**Issue by barracuda156** [dbohdan/initool#17]
@dbohdan Something broke down in 0.17.0 as compared to 0.15.0:
```
--->  Configuring initool
--->  Building initool
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-0.17.0" && /usr/bin/make -j6 -w all CC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cc/usr/bin/gcc-4.2" CXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cxx/usr/bin/g++-4.2" OBJC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objc/usr/bin/gcc-4.2" OBJCXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objcxx/usr/bin/g++-4.2" INSTALL="/usr/bin/install -c" 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-0.17.0'
mlton initool.mlb
sh test.sh
tests/comments-2.command: line 1: ./initool: No such file or directory
tests/comments-2.command: line 3: ./initool: No such file or directory
tests/comments-2.command: line 5: ./initool: No such file or directory
tests/comments-2.command: line 7: ./initool: No such file or directory
make: *** [test] Error 127
make: *** Waiting for unfinished jobs....
make: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-0.17.0'
```

I can confirm that 0.15.0 builds fine with everything else identical.

**Comment by dbohdan** [comment]
I cannot replicate this without MacPorts, but I have an idea what the cause may be. Try building https://github.com/dbohdan/initool/commit/39af92716115bf36140776cda119d20286c2d493 (currently the latest commit on the branch `dev`). If it works, I'll release version 0.17.1.

**Comment by barracuda156** [comment]
While it still failed initially:
```
--->  Building initool
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-39af92716115bf36140776cda119d20286c2d493" && /usr/bin/make -j6 -w all CC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cc/usr/bin/gcc-4.2" CXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cxx/usr/bin/g++-4.2" OBJC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objc/usr/bin/gcc-4.2" OBJCXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objcxx/usr/bin/g++-4.2" INSTALL="/usr/bin/install -c" 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-39af92716115bf36140776cda119d20286c2d493'
mlton initool.mlb
sh test.sh
tests/comments-2.command: line 1: ./initool: No such file or directory
tests/comments-2.command: line 3: ./initool: No such file or directory
tests/comments-2.command: line 5: ./initool: No such file or directory
tests/comments-2.command: line 7: ./initool: No such file or directory
make: *** [test] Error 127
make: *** Waiting for unfinished jobs....
```
I figured out how to fix it – disable parallel build. With that it worked:
```
--->  Building initool
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-39af92716115bf36140776cda119d20286c2d493" && /usr/bin/make -w all CC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cc/usr/bin/gcc-4.2" CXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/cxx/usr/bin/g++-4.2" OBJC="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objc/usr/bin/gcc-4.2" OBJCXX="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/compwrap/objcxx/usr/bin/g++-4.2" INSTALL="/usr/bin/install -c" 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-39af92716115bf36140776cda119d20286c2d493'
mlton initool.mlb
sh test.sh
./initool version > VERSION
make: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_initool/initool/work/initool-39af92716115bf36140776cda119d20286c2d493'
--->  Staging initool into destroot
```

**Comment by barracuda156** [comment]
@dbohdan In fact 0.17.0 also works once parallel building is disabled. Closing, thank you.

**Comment by dbohdan** [comment]
Great!

**Comment by barracuda156** [comment]
Updated in https://github.com/macports/macports-ports/pull/24200/commits/7d92f9eba88f47eb43b49b89e361e92d4379c1f8

**Comment by dbohdan** [comment]
I think I have fixed the problem in 53cb8d03378ba67db7646b376e0641118d96dfd7. See if you can re-enable parallel build with this commit.
